Tests fail on Linux

When I try to run the tests on linux, I get an error mentionning macOS. Is there something to configure for linux ?

```
$ npm run test
> hyperterm@1.0.0 test /home/benoit/Documents/Perso/HyperTerm
> npm run pack && xo && ava

> hyperterm@1.0.0 pack /home/benoit/Documents/Perso/HyperTerm
> npm run build && build --dir && babel --no-comments --compact --minified --out-file app/dist/bundle.js app/dist/bundle.js

> hyperterm@1.0.0 build /home/benoit/Documents/Perso/HyperTerm
> NODE_ENV=production webpack

Hash: 8c8d93c75a36f6fb1f15
Version: webpack 2.1.0-beta.22
Time: 2846ms
           Asset     Size  Chunks             Chunk Names
       bundle.js  1.57 MB       0  [emitted]  main
   bundle.js.map  1.93 MB       0  [emitted]  main
assets/icons.svg  1.07 kB          [emitted]  
    + 281 hidden modules
Rebuilding app dependencies for arch x64 to /home/benoit/Documents/Perso/HyperTerm/app

Packaging for platform linux x64 using electron 1.4.1 to dist/linux-unpacked


   1 failed


   1. before
   failed with "ENOENT: no such file or directory, stat '/home/benoit/Documents/Perso/HyperTerm/dist/mac/HyperTerm.app/Contents/MacOS/HyperTerm'"
  
npm ERR! Linux 3.16.0-4-amd64
npm ERR! argv "/home/benoit/.nvm/versions/node/v6.5.0/bin/node" "/home/benoit/.nvm/versions/node/v6.5.0/bin/npm" "run" "test"
npm ERR! node v6.5.0
npm ERR! npm  v3.10.3
npm ERR! code ELIFECYCLE
npm ERR! hyperterm@1.0.0 test: `npm run pack && xo && ava`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the hyperterm@1.0.0 test script 'npm run pack && xo && ava'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the hyperterm package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     npm run pack && xo && ava
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs hyperterm
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls hyperterm
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/benoit/Documents/Perso/HyperTerm/npm-debug.log
```

--- a/test/index.js
+++ b/test/index.js
@@ -8,8 +8,23 @@ import {Application} from 'spectron';
 let app;
 
 test.before(async () => {
+  let pathToBinary;
+
+  switch (process.platform) {
+    case 'linux':
+      pathToBinary = path.join(__dirname, '../dist/linux-unpacked/HyperTerm');
+      break;
+
+    case 'darwin':
+      pathToBinary = path.join(__dirname, '../dist/mac/HyperTerm.app/Contents/MacOS/HyperTerm');
+      break;
+
+    default:
+      throw new Error('Path to the built binary needs to be defined for this platform in test/index.js');
+  }
+
   app = new Application({
-    path: path.join(__dirname, '../dist/mac/HyperTerm.app/Contents/MacOS/HyperTerm')
+    path: pathToBinary
   });
 
   await app.start();

